scroll_ui/Scripts/ButtonsBehaviours/AlbumBehaviour.gd

```diff
@@ -3,7 +3,7 @@ extends Button
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	pass # Replace with function body.
+	pass  # Replace with function body.
```

Comment:
Is this double space a thing? This comment is autogenerated by Godot, you'd think it's the correct way to do it.